tools/wptrunner/wptrunner/browsers/content_shell.py

```diff
@@ -63,8 +63,12 @@ def browser_kwargs(logger, test_type, run_info_data, config, **kwargs):
 
 def executor_kwargs(logger, test_type, test_environment, run_info_data,
                     **kwargs):
+    sanitizer_enabled = kwargs.get("sanitizer_enabled")
+    if sanitizer_enabled:
+        test_type = "crashtest"
```

Comment:
I think we are not using "sanitizer_enabled" for chrome, so do not need to make this change for chrome yet?